Source/Core/CommandLineOptions.cs

```diff
@@ -1675,6 +1677,10 @@ public int GetArgumentSeparatorIndex(string argList, int startIndex)
     {:ignore}
       Ignore the declaration (after checking for duplicate names).
 
+    {:exclude_dep}
+      Ignore the declaration for the purpose of pruning, i.e., do not consider
```

Comment:
What is the purpose?  The description is mysterious and does not make sense even after reading the PR description.

Reply:
Added this to the PR description